src/redux/actions/game.ts

```diff
@@ -1,16 +1,15 @@
-import { ActionsUnion } from './type-helpers';
-
 import { Play } from '../../game-engine/positions';
 import Move from '../../game-engine/Move';
 import { State } from '../../game-engine/application-states';
 
+
 export enum GameActionType {
   CHOOSE_OPPONENT = 'GAME.CHOOSE_OPPONENT',
   CHOOSE_A_PLAY = 'GAME.CHOOSE_A_PLAY',
   EVENT_RECEIVED = 'GAME.EVENT_RECEIVED',
   MOVE_RECEIVED = 'GAME.MOVE_RECEIVED',
   MOVE_SENT = 'GAME.MOVE_SENT',
-  STATE_CHANGED = 'GAME.STATE_CHANGED',
+  STATE_CHANGED = 'GAME.STATE_CHANGED'
```

Comment:
Currently, our ts-lint is not set up to enforce this (otherwise it would fail to build). I can add the rule `"trailing-comma": [true, {"multiline":{"objects": "always"}}]` to tslint.config to enforce this.